pom.xml

```diff
@@ -179,6 +180,11 @@
     <!-- Version of the KIE Workbench application. Shown for example in About dialog. Will be usually overridden by productisation. -->
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
+    <version.wildfly.swarm>2016.9</version.wildfly.swarm>
+    <version.net.byte-buddy>1.4.16</version.net.byte-buddy>
+    <version.commons-beanutils>1.8.3</version.commons-beanutils>
+    <version.docker-client>3.5.12</version.docker-client>
+    <version.arquillian-cube>1.0.0.Alpha15</version.arquillian-cube>
```

Comment:
@Salaboy could you please also create a PR for ip-bom adding these deps? Once it would get merged, and we would upgrade the ip-bom version, then we could remove these...Thanks!